packages/lending/src/protocols/aave-v2/configs.ts

```diff
@@ -109,128 +34,30 @@ export const configs: Config[] = [
-      },
-    ],
   },
 ];
 
-export const [
-  supportedChainIds,
-  configMap,
-  reserveMap,
-  tokensForDepositMap,
-  aTokensForDepositMap,
-  tokensForWithdrawMap,
-  aTokensForWithdrawMap,
-  tokensForBorrowMap,
-  tokensForRepayMap,
-  tokensForFlashLoanMap,
-] = configs.reduce(
-  (accumulator, config) => {
-    const { chainId, reserves } = config;
-
-    accumulator[0].push(chainId);
-    accumulator[1][chainId] = config;
-    accumulator[2][chainId] = {};
-    accumulator[3][chainId] = [];
-    accumulator[4][chainId] = [];
-    accumulator[5][chainId] = [];
-    accumulator[6][chainId] = [];
-    accumulator[7][chainId] = [];
-    accumulator[8][chainId] = [];
-    accumulator[9][chainId] = [];
-    for (const reserve of reserves) {
-      const { asset, aToken, used } = reserve;
-      if (!asset.isNative) {
-        accumulator[2][chainId][asset.address] = reserve;
-        if (aToken) accumulator[2][chainId][aToken.address] = reserve;
-      }
-
-      if (used.deposit) {
-        accumulator[3][chainId].push(asset);
-        if (aToken) accumulator[4][chainId].push(aToken);
-      }
-      if (used.withdraw) {
-        accumulator[5][chainId].push(asset);
-        if (aToken) accumulator[6][chainId].push(aToken);
-      }
-      if (used.borrow) accumulator[7][chainId].push(asset);
-      if (used.repay) accumulator[8][chainId].push(asset);
-      if (used.flashLoan) accumulator[9][chainId].push(asset);
-    }
-    accumulator[6][chainId] = Array.from(new Set(accumulator[6][chainId])); // uniq aToken
-
-    return accumulator;
-  },
-  [[], {}, {}, {}, {}, {}, {}, {}, {}, {}] as [
-    number[],
-    Record<number, Config>,
-    Record<number, Record<string, Reserve>>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>
-  ]
-);
+export const supportedChainIds = logics.aavev2.supportedChainIds;
 
 export function getContractAddress(chainId: number, name: ContractName) {
-  return configMap[chainId].contractMap[name];
+  const { contractMap } = configs.find((configs) => configs.chainId === chainId)!;
+  return contractMap[name];
 }
 
-export function toAToken(chainId: number, token: common.Token) {
-  return reserveMap[chainId][token.wrapped.address].aToken;
-}
+const depositDisableMap: Record<number, string[]> = {
+  [common.ChainId.mainnet]: [],
+  [common.ChainId.polygon]: [],
+};
 
-export function isAToken(chainId: number, token: common.Token): boolean {
-  const aToken = reserveMap[chainId][token.address].aToken;
-  return aToken.is(token);
-}
+const borrowDisableMap: Record<number, string[]> = {
+  [common.ChainId.mainnet]: [common.mainnetTokens.AAVE.address, common.mainnetTokens.stETH.address],
+  [common.ChainId.polygon]: [common.polygonTokens.AAVE.address],
+};
```

Comment:
```suggestion
  [common.ChainId.mainnet]: [common.mainnetTokens.stETH.address],
  [common.ChainId.polygon]: [],
  [common.ChainId.avalanche]: [],
};
```

`AAVE` is not in `service.getReserveTokens()`

Reply:
removed